tests/test_data_preparation.py

```diff
@@ -24,25 +34,50 @@ def compute_md5(file_path):
     return hash_md5.hexdigest()
 
 
+def docker_rm_and_mkdir(file_):
+    directory = Path(file_).absolute().parent
+    test_config_path = Path(__file__).absolute().parent / "gpu-tests" / "test-local.yaml"
+    config = yaml.safe_load(open(test_config_path).read())
+    volumes = config['mounts']
+    container = config['containers']['nemo-skills']
+    rm_cmd = f"rm -f {str(file_)}"
+    mkdir_cmd = f"mkdir -p {str(directory)}"
+    docker_run(
```

Comment:
Probably a bit more efficient to run ones with a joint command

Reply:
done in 126e43d2f87616d0ecda2bdd1fda544ebf9af314